utils/S3UtilsMongoClient.js

```diff
@@ -269,6 +374,138 @@ class S3UtilsMongoClient extends MongoClientInterface {
             return cb();
         });
     }
+
+    updateStorageConsumptionMetrics(countItems, dataMetrics, log, cb) {
+        const updatedStorageMetricsList = [{ _id: __COUNT_ITEMS, value: countItems }];
+        // iterate every resource through dataMetrics and add to updatedStorageMetricsList
+        Object.keys(dataMetrics).forEach(metricLevel => {
+            if (validStorageMetricLevels.has(metricLevel)) {
+                Object.keys(dataMetrics[metricLevel]).forEach(resource => {
+                    updatedStorageMetricsList.push({
+                        _id: `${metricLevel}:${resource}`,
```

Comment:
```suggestion
                        _id: `${metricLevel}_${resource}`,
```

for consistency with the design (unless you found an issue of using `_`)

Reply:
for the the existing bucketList in _countitems document
seems it is used by [pensieve-api](https://github.com/scality/pensieve-api/blob/development/1.4/apiv1/restapi/status/item_counts_stats.go), but just I don't know where they are presented on the UI